Grbl_Esp32/src/CoolantControl.h

```diff
@@ -23,24 +23,17 @@
   along with Grbl.  If not, see <http://www.gnu.org/licenses/>.
 */
 
```

Comment:
I worried about that too - but since everything flows from GCode, anything that is a representation of a GCode state also makes sense in GCode.h.  So I stopped worrying.